### PR TITLE
Spam us with open discovery PRs

### PR DESCRIPTION
**Pull request by gstamp**
image

/cc @envato/discovery @johnsyweb

### CODE REVIEW

**Comment by salamagd** [comment]
:+1:

**Comment by elseano** [comment]
:+1:

File: config/envato.yml [hunk 1]
```diff
@@ -17,5 +17,21 @@ envato-market:
   exclude_labels:
     - wip
     - WIP
-
-
+discovery:
+  channel: '#discovery-notify'
+  members: []
+  repos:
```
Comment (simpsora) [inline config/envato.yml]: How about also:
* `ami-builder`
* `discovery-docker`
* `recommender-importer-exporter-whatever-its-called`

**Comment by johnsyweb** [comment]
Hmmm...
```
[Tue 15/08/11 09:56 AEST][s001][x86_64/darwin14.3.0/14.4.0][5.0.8]
<paj@clacks:~/src/seal>
zsh 6862  (git)-[discovery]-% ./bin/informative_seal.rb discovery
/Users/paj/src/seal/lib/github_fetcher.rb:54:in `labels': undefined method `number' for [:message, "Moved Permanently"]:Array (NoMethodError)
	from /Users/paj/src/seal/lib/github_fetcher.rb:64:in `hidden_labels'
	from /Users/paj/src/seal/lib/github_fetcher.rb:59:in `hidden?'
	from /Users/paj/src/seal/lib/github_fetcher.rb:24:in `block (2 levels) in list_pull_requests'
	from /opt/rubies/2.1.2/lib/ruby/gems/2.1.0/gems/sawyer-0.6.0/lib/sawyer/resource.rb:125:in `each'
	from /opt/rubies/2.1.2/lib/ruby/gems/2.1.0/gems/sawyer-0.6.0/lib/sawyer/resource.rb:125:in `each'
	from /Users/paj/src/seal/lib/github_fetcher.rb:24:in `reject'
	from /Users/paj/src/seal/lib/github_fetcher.rb:24:in `block in list_pull_requests'
	from /Users/paj/src/seal/lib/github_fetcher.rb:22:in `each'
	from /Users/paj/src/seal/lib/github_fetcher.rb:22:in `list_pull_requests'
	from /Users/paj/src/seal/lib/seal.rb:40:in `pull_requests'
	from /Users/paj/src/seal/lib/seal.rb:19:in `bark'
	from ./bin/informative_seal.rb:5:in `<main>'
```
Looks like we need more logging!

**Comment by joneslee85** [comment]
:+1:

File: config/envato.yml (hunk 1 above)
Comment (gstamp) [inline config/envato.yml]: recommender-importer-exporter-whatever-its-called is obsolete but I'll add the others.

**Comment by gstamp** [comment]
Currently blocked because it's not handling github api redirect responses correctly.

Will come back to this when I have a bit of time.

**Comment by gstamp** [comment]
Happy for this to be merged @johnsyweb ?

**Comment by johnsyweb** [comment]
:shipit:

**Pull request merged**